### PR TITLE
AdmixerBidAdapter - add alias , add adapter config

### PR DESCRIPTION
**Pull request by Galphimbl**
## Type of change
- [x] Feature

## Description of change

add alias
add "endpoint_url" bidder config

- A link to a PR on the docs repo [prebid.github.io#2572](https://github.com/prebid/prebid.github.io/pull/2572)

### CODE REVIEW

File: modules/adblender.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
+# Overview
+
+Module Name: AdBlender Bidder Adapter  
+Module Type: Bidder Adapter  
+Maintainer: contact@ad-blender.com
+
+# Description
+
+Connects to Go2Net demand source to fetch bids.
```
Comment (osazos) [inline modules/adblender.md:9]: Should not be "Ad blender" ?

**Review by osazos — COMMENTED** [review]
Hi, 

In the adBlender.md file you have a section about the custom endpoint. Is this section shouldn't be in the admixerAdapter.md too?

File: modules/adblender.md (hunk 1 above)
Comment (Galphimbl) [reply modules/adblender.md]: yeap, fixed.

**Comment by Galphimbl** [comment]
Hi.
no it's only for adblender.md. it should work, but officially supported only by adblender

**Review by osazos — APPROVED** [review]
LGTM

**Pull request merged**